content/docs/1.4.0/deploy/uninstall/_index.md

```diff
@@ -65,6 +70,10 @@ pressing `Ctrl C` then running `kubectl create -f https://raw.githubusercontent.
 
 
 ### Troubleshooting
+#### Uninstalling using Rancher UI or Helm failed, I am not sure why
+You might want to check the logs of the `longhorn-uninstall-xxx` pod inside `longhorn-system` namespace to see why it failed.
+One reason can be that [deleting-confirmation-flag](../../references/settings/#deleting-confirmation-flag) is `false`.
```

Comment:
Suggest letting users know how to continue when having this issue. As you mentioned, they can just simply rerun uninstall or rollback. 

ref: https://github.com/longhorn/longhorn/pull/4712#discussion_r994939541

Reply:
Thank you. Fixed